### DISCUSSION TITLE
Discussion post permissions bug

### DISCUSSION THREAD

**Discussion by indigo-jay** [community/community#10827] [Discussions]
According to the docs, a person associated with an organization with "member" role should not be able to edit discussion posts from others. Unfortunately, they can. My organization has a few people with member role, who are in a team. In the Discussion tab for the team, any member of the team regardless of role can edit any discussion post.

**Comment by volkantash** [comment]
Other same problem
https://github.com/github/feedback/discussions/10447

**Comment by indigo-jay** [comment]
Someone from Github support replied to my ticket that this was expected behavior for team discussions, which basically have no permissions like repo discussions do. He said he would escalate a feature request. My opinion is that both discussion areas should follow the same permission structure, but that's left up to them to decide.
